x-pack/plugins/infra/public/components/range_date_picker/index.tsx

```diff
@@ -29,14 +31,144 @@ import {
 } from '@elastic/eui';
 
 const commonDates = [
-  'Today',
-  'Yesterday',
-  'This week',
-  'Week to date',
-  'This month',
-  'Month to date',
-  'This year',
-  'Year to date',
+  {
+    id: 'Today',
```

Comment:
Let's make id value more similar to id:
```suggestion
    id: 'today',
```